installer/conf/omsagent.d/wlm_baseOS.conf

```diff
@@ -55,3 +55,62 @@
   tag oms.wlm.monitor
 </source>
 
+<source>
```

Comment:
Shouldn't the old <source> be removed?...this conf will emit perf data twice (one for each <source> tag.)

Reply:
The old source tags will be removed once the support for the old data type (which is the only difference between the two packets) is removed from the service.